mPowerSDK/Dashboard/APHCorrelationsSelectorViewController.m

```diff
@@ -0,0 +1,203 @@
+//
+//  APHCorrelationsSelectorViewController.m
+//  mPowerSDK
+//
+// Copyright (c) 2015, Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+// 1.  Redistributions of source code must retain the above copyright notice, this
+// list of conditions and the following disclaimer.
+//
+// 2.  Redistributions in binary form must reproduce the above copyright notice,
+// this list of conditions and the following disclaimer in the documentation and/or
+// other materials provided with the distribution.
+//
+// 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+// may be used to endorse or promote products derived from this software without
+// specific prior written permission. No license is granted to the trademarks of
+// the copyright holders even if such marks are included in this software.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+#import "APHCorrelationsSelectorViewController.h"
+#import "APHLocalization.h"
+#import "APHScoring.h"
+
+#import <APCAppCore/UIColor+APCAppearance.h>
+
+@interface APHCorrelationsSelectorViewController ()
+@property (strong, nonatomic) APHScoring *scoring;
+@property (strong, nonatomic) NSArray *scoringObjects;
+
+@property (strong, nonatomic) UITableView *tableView;
+@property (nonatomic) CGRect downFrame;
+@property (nonatomic) CGRect upFrame;
+@end
+
+@implementation APHCorrelationsSelectorViewController 
+
+- (id)initWithScoringObjects:(NSArray *)scoringObjects
+{
+    self = [super init];
+    if (self) {
+        self.view.backgroundColor = [[UIColor blackColor] colorWithAlphaComponent:0.8];
+        self.scoringObjects = scoringObjects;
+        
+        [self initTableView];
+        [self.tableView registerClass:[UITableViewCell class] forCellReuseIdentifier:@"Cell"];
```

Comment:
Wow, totally surprised this isn't crashing. Do *NOT* reference self.view or self.table in initializer of a view controller. Put this in viewDidLoad.